randovania/games/am2r/exporter/patch_data_factory.py

```diff
@@ -146,12 +149,12 @@ def _create_pickups_dict(self, pickup_list: list[ExportedPickupDetails], item_in
             pickup_map_dict[object_name] = {
                 "sprite_details": {
                     "name": pickup.model.name,
-                    "speed": item_info[pickup.original_pickup.name]["sprite_speed"],
+                    "speed": model_data[pickup.model.name] if model_data.get(pickup.model.name) is not None else 0.2,
```

Comment:
`"speed": model_data.get(pickup.model.name, 0.2),`

Reply:
ah, wasnt aware that existed. thanks